cli/tests/test_git_push.rs

```diff
@@ -47,13 +47,13 @@ fn set_up(test_env: &TestEnvironment) {
         .success();
     origin_dir.run_jj(["git", "export"]).success();
 
+    test_env.add_config("remotes.origin.auto-track-bookmarks = 'glob:*'");
```

Comment:
Can you move this to callers? I think it will help rewrite tests in a way that they don't depend on auto tracking.